dir.targets

```diff
@@ -154,5 +154,12 @@
 
   </Target>
 
+  <Target Name="SetPublicReleaseVersion" AfterTargets="GetBuildVersion">
+    <PropertyGroup Condition=" '$(PublicRelease)' == 'true' ">
+      <!-- Removes the prerelease tag from version when this is a public version but still has the git commit-->
+      <AssemblyInformationalVersion>$(BuildVersionSimple)$(SemVerBuildSuffix)</AssemblyInformationalVersion>
```

Comment:
An interesting perspective here. If what you're building is a prerelease, why wouldn't you want that to show up in your AssemblyInformationalVersion? Wouldn't it be useful for you or a customer to see a DLL and associate it with -beta or -rc or recognize that since it's missing it must be at or near RTW?